File: Zestawy Zadań/C++/4 - Tablice/Zestaw 2 zadania/4.2.2 - Tablica 12 liczb calkowitch i wypisywanie ile razy Y w tablicy.cpp
#include <iostream>

using namespace std;

int main()
{
    int tab[12];
    int Y = 0;
    int licznik = 0;
    for (int i = 0; i < 12; i++)
    {
        cin >> tab[i];
    }
    cout << "Podaj Y: " << endl;
    cin >> Y;
    for (int i = 0; i < 12; i++)
    {
        if (Y == tab[i])
        {
            licznik++;
        }
    }
    cout << "Liczba " << Y << " znajduje sie "<< licznik << " razy w tablicy." << endl;
    return 0;
}

File: Zestawy Zadań/Python/4 - Tablice/Zestaw 2 zadania/4.2.2 - Tablica 12 liczb calkowitch i wypisywanie ile razy Y w tablicy.py
tab = [0] * 12
Y = 0
licznik = 0

for i in range(0, 12):
    tab[i] = int(input("Podaj liczby: "))

Y = int(input("Podaj Y: "))

for i in range(0, 12):
    if tab[i] == Y:
        licznik += 1

print("Liczba", Y, "znajduje się", licznik, "raz(y) w liście.")